routing_model/src/scenario_tests.rs

```diff
@@ -39,6 +39,7 @@ const ADD_PROOFING_NODE_1: NodeChange =
     NodeChange::AddResourceProofing(Node(Attributes { name: 1, age: 10 }));
 const SET_ONLINE_NODE_1: NodeChange = NodeChange::Online(Node(Attributes { name: 1, age: 10 }));
 const REMOVE_NODE_1: NodeChange = NodeChange::Remove(Node(Attributes { name: 1, age: 10 }));
+const SET_OFFLINE_NODE_130: NodeChange = NodeChange::Offline(Node(Attributes { name: 130, age: 30 }));
```

Comment:
This is more because it is used a lot, and all these are about NODE_1.
You're probably better off using `NodeChange::Offline(NODE_ELDER_130.clone())` directly in place.

Reply:
clone is probably not necessary in my suggestion.

Reply:
Yep it wasn't necessary